solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -910,9 +910,7 @@ contract LiquidityProtection is ILiquidityProtection, Utils, Owned, ReentrancyGu
             );
         }
 
-        ILiquidityPoolConverter converter = ILiquidityPoolConverter(payable(ownedBy(poolToken)));
-        IERC20 otherReserve = converterOtherReserve(converter, reserveToken);
-        (uint256 rateN, uint256 rateD) = converterReserveBalances(converter, otherReserve, reserveToken);
+        (uint256 rateN, uint256 rateD, , ) = reserveTokenRates(poolToken, reserveToken, true);
```

Comment:
I noticed reserveTokenRates doc doesn't describe the return values, can you add it?

Reply:
done